.github/workflows/build-and-publish-PyPI.yml

```diff
@@ -4,8 +4,8 @@ name: Build and Publish to PyPI
 
 on:
   push:
-    branches:
-      - 'main'
+    tags:
+      - '*'
 
 jobs:
   build-and-publish:
```

Comment:
`python-version: '3.10'` are we using this version in sdk?

Reply:
Our setup.py specifies python >= 3.6. I used that as a default value, although I am using 3.9 for development. I am not sure which version @sjpark608 is using.

Reply:
@sjpark608 how did we come up w/ 3.6?